File: Kritika Tripathi/Basics Of Programming/Patterns/Pattern6.cpp
#include <iostream>
using namespace std;

int main(){
    int n=5,j;
    int scale = n/2+1;

    //write your code here
    for(int i=1; i<=scale; i++){
        for(j=1; j<=scale+1-i; j++){
            cout<<"*"<<"\t";
        }
        for(; j<=scale+i; j++){
            cout<<"\t";
        }
        for(;j<=2*scale+1;j++){
            cout<<"*\t";
        }
        cout<<endl;
    }
    for(int i=n; i>scale; i--){
        for(j=1; j<=scale+1-(i-scale); j++){
            cout<<"*"<<"\t";
        }
        for(; j<=i; j++){
            cout<<"\t";
        }
        for(;j<=2*scale+1;j++){
            cout<<"*\t";
        }
        cout<<endl;
    }
    return 0;
}

File: V.Vennela/Basics of Programming/Patterns/Pattern6.java
import java.util.*;

public class Pattern6 {

    public static void main(String[] args) {
        Scanner scn = new Scanner(System.in);

        // write ur code here
        int n = scn.nextInt();
    int st = n / 2 + 1, sp = 1; 
    for (int i = 1; i <= n; i++)
    {
      for (int j = 1; j <= st; j++) 
      {
        System.out.print("*	");
      }
      for (int j = 1; j <= sp; j++) 
      {
        System.out.print("	");
      }
      for (int j = 1; j <= st; j++) 
      {
        System.out.print("*	");
      }

      if ( i <= n / 2)
      { 
          st--;  
          sp+=2; 
      }
        else
        {
          st++; 
          sp-=2; 
            
        }
          System.out.println();   
        }
      

    }
}
